app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -242,6 +246,9 @@ private void updateCurrentBgInfoForBtBasedWixel(TextView notificationText) {
     }
 
     private void updateCurrentBgInfoCommon(TextView notificationText) {
+        if (alreadyDisplayedBgInfoCommon) return; // with bluetooth and wifi, skip second time
```

Comment:
It is set to false again in line 208 with comment "reset flag" in updateCurrentBgInfo()

I thought having a flag/state variable made it more future proof rather than depending on conditional logic based on the collection mode.

The reason for not displaying the data a second time is because it has already been shown once. It isn't just because it is in the Wifi+Bluetooth collection mode, although that is the only place it can occur today. I think either approach is valid.